spec/index.bs

```diff
@@ -6096,8 +6103,8 @@ called the render pass encoder can no longer be used.
                         Issue: Add remaining validation.
                     </div>
 
-                1. Push each command in |this|.{{GPUProgrammablePassEncoder/[[command_list]]}} onto the end of
-                    |this|.{{GPUProgrammablePassEncoder/[[command_encoder]]}}.{{GPUCommandEncoder/[[command_list]]}}.
+                1. [=list/Extend=] |this|.{{GPUProgrammablePassEncoder/[[command_encoder]]}}.{{GPUCommandEncoder/[[command_list]]}}
+                    with |this|.{{GPUProgrammablePassEncoder/[[command_list]]}}.
```

Comment:
Somewhere we need to talk about what happens at the end of a pass (attachment stores and resolves), should leave a TODO.